src/libasr/pass/pass_utils.cpp

```diff
@@ -990,8 +990,12 @@ namespace LCompilers {
             for (size_t i=0; i<loop.n_body; i++) {
                 body.push_back(al, loop.m_body[i]);
             }
+            Vec<ASR::stmt_t*> orelse;
+            orelse.reserve(al, loop.n_orelse);
+            for (size_t i = 0; i < loop.n_orelse; i++)
+                orelse.push_back(al, loop.m_orelse[i]);
             ASR::stmt_t *while_loop_stmt = ASRUtils::STMT(ASR::make_WhileLoop_t(al, loc,
-                loop.m_name, cond, body.p, body.size()));
+                loop.m_name, cond, body.p, body.size(), orelse.p, orelse.size()));
```

Comment:
Feels redundant. `orelse` being pushed to a new memory location. Nothing else I think.